internal/drivers/basic/account_management.go

```diff
@@ -0,0 +1,137 @@
+/******************************************************************************
```

Comment:
Documentation for this driver should be added to `docs/drivers`. Please take a look at the existing driver docs for reference.